deb/publish.go

```diff
@@ -452,6 +454,36 @@ func (p *PublishedRepo) GetLabel() string {
 	return p.Label
 }
 
+// GetAppStream returns full path list of appstream files
+func (p *PublishedRepo) GetAppStream(component string) ([]string, error) {
+	if utils.Config.AppStreamDir == "" {
```

Comment:
The `Publish` function below will call the `GetAddonFiles` method. Should I pass the `AppStreamDir` all the way down from the API and CLI calls?